src/db.ts

```diff
@@ -10,8 +10,27 @@ export const connect = async () => {
       useFindAndModify: false,
       useNewUrlParser: true,
     });
-  } catch (error) {
-    logger.error("Error while connecting to mongo database", error);
+  } catch (error: any) {
+    const errorMessage = error.toString();
+    if (errorMessage.includes("ECONNREFUSED")) {
+      logger.error("\n\n\n\x1b[1m\x1b[31m%s\x1b[0m", error);
+      console.log(
```

Comment:
Logger is used to avoid using console logs. You're console logging inside the logger itself.

Reply:
@xoldyckk I understand your point but I have used logger to log error and for rest of fixes I used console.log because it was looking better. I can restore it if you want.

Reply:
@aviraldevv You should opt for a solution that fixes the message formatting of logger itself instead of opting to use console log to make it easier for contributors to easily display that something went wrong with mongodb.

Reply:
@aviraldevv Yes. The task you're given is format those mongoose errors to be easily readable, maybe even add some custom stuff. But you have to do that with logger itself. You can't use console.log.

Reply:
@xoldyckk Please check slack.